Binary files qpdfview-0.4.16/objects/mainwindow.o and qpdfview-0.4.16-roand/objects/mainwindow.o differ
Binary files qpdfview-0.4.16/qpdfview and qpdfview-0.4.16-roand/qpdfview differ
diff -cr qpdfview-0.4.16/sources/mainwindow.cpp qpdfview-0.4.16-roand/sources/mainwindow.cpp
*** qpdfview-0.4.16/sources/mainwindow.cpp	2015-11-16 23:23:23.180090140 +0300
--- qpdfview-0.4.16-roand/sources/mainwindow.cpp	2015-11-14 23:48:59.000000000 +0300
***************
*** 2896,2902 ****
  
      // progress and error icons
  
!     s_settings->pageItem().setProgressIcon(QIcon());
      s_settings->pageItem().setErrorIcon(loadIconWithFallback(QLatin1String("image-missing")));
  }
  
--- 2896,2902 ----
  
      // progress and error icons
  
!     s_settings->pageItem().setProgressIcon(loadIconWithFallback(QLatin1String("image-loading")));
      s_settings->pageItem().setErrorIcon(loadIconWithFallback(QLatin1String("image-missing")));
  }
  
Only in qpdfview-0.4.16/sources: mainwindow.cpp.orig
diff -cr qpdfview-0.4.16/translations/qpdfview_ru.ts qpdfview-0.4.16-roand/translations/qpdfview_ru.ts
*** qpdfview-0.4.16/translations/qpdfview_ru.ts	2015-11-14 23:48:59.000000000 +0300
--- qpdfview-0.4.16-roand/translations/qpdfview_ru.ts	2015-11-16 23:17:59.000000000 +0300
***************
*** 622,638 ****
      <message>
          <location filename="../sources/mainwindow.cpp" line="2731"/>
          <source>Match &amp;case</source>
!         <translation type="unfinished">С уч&amp;ётом регистра</translation>
      </message>
      <message>
          <location filename="../sources/mainwindow.cpp" line="2732"/>
          <source>Whole &amp;words</source>
!         <translation type="unfinished">Слова &amp;целиком</translation>
      </message>
      <message>
          <location filename="../sources/mainwindow.cpp" line="2733"/>
          <source>Highlight &amp;all</source>
!         <translation type="unfinished">Выделить &amp;все</translation>
      </message>
      <message>
          <location filename="../sources/mainwindow.cpp" line="2795"/>
--- 622,638 ----
      <message>
          <location filename="../sources/mainwindow.cpp" line="2731"/>
          <source>Match &amp;case</source>
!         <translation type="unfinished">Регистр</translation>
      </message>
      <message>
          <location filename="../sources/mainwindow.cpp" line="2732"/>
          <source>Whole &amp;words</source>
!         <translation type="unfinished">Слова</translation>
      </message>
      <message>
          <location filename="../sources/mainwindow.cpp" line="2733"/>
          <source>Highlight &amp;all</source>
!         <translation type="unfinished">Все</translation>
      </message>
      <message>
          <location filename="../sources/mainwindow.cpp" line="2795"/>
